packages/flutter/lib/src/gestures/scale.dart

```diff
@@ -145,23 +152,42 @@ class ScaleUpdateDetails {
   /// Expressed in radians.
   final double rotation;
 
+  /// The number of pointers being tracked by the gesture recognizer.
+  ///
+  /// Typically this is the number of fingers being used to pan the widget using the gesture
+  /// recognizer.
+  final int pointerCount;
+
   @override
-  String toString() => 'ScaleUpdateDetails(focalPoint: $focalPoint, localFocalPoint: $localFocalPoint, scale: $scale, horizontalScale: $horizontalScale, verticalScale: $verticalScale, rotation: $rotation)';
+  String toString() => 'ScaleUpdateDetails('
+    'focalPoint: $focalPoint,'
+    ' localFocalPoint: $localFocalPoint,'
+    ' scale: $scale,'
+    ' horizontalScale: $horizontalScale,'
+    ' verticalScale: $verticalScale,'
+    ' rotation: $rotation,'
+    ' pointerCount: $pointerCount)';
 }
 
 /// Details for [GestureScaleEndCallback].
 class ScaleEndDetails {
   /// Creates details for [GestureScaleEndCallback].
   ///
   /// The [velocity] argument must not be null.
-  ScaleEndDetails({ this.velocity = Velocity.zero })
+  ScaleEndDetails({ this.velocity = Velocity.zero, this.pointerCount = 0, })
```

Comment:
```suggestion
  ScaleEndDetails({ this.velocity = Velocity.zero, this.pointerCount = 0 })
```

I've searched around the repo and have found 2 use cases of `, })`. It's safe to assume to be a typo.